stdlib/privacy/src/main/ballerina/privacy/database_pii_store_utils.bal

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# Represents personally identifiable information (PII) stored in a database
+#
+# + pii - personally identifiable information
+type PiiData record {
+    string pii;
+};
+
+# Build insert query based on the table name and column names
+#
+# + tableName - table name used to store PII
+# + idColumn - column name used to store pseudonymized identifier
+# + piiColumn - column name used to store PII
+# + return - insert query
+function buildInsertQuery (string tableName, string idColumn, string piiColumn) returns string {
+    return "INSERT INTO " + tableName + " (" + idColumn + "," + piiColumn + ") VALUES (?, ?)";
```

Comment:
We have io:sprintf function. Can we utilize that in here? WDYT?

Reply:
+1. Will update.